README.md

```diff
@@ -1,161 +1,245 @@
-# hello-microverse
-This is just basic min-project to must github flow 
 
- 
- ![murple_logo](https://user-images.githubusercontent.com/110924992/218853728-1a8f5726-cd6a-4613-85ed-38dd5132cd9f.png)
- 
- 
- 
- 
 
- 
-Microverse README Template
-📗 Table of Contents
+hello microverse
 
-📖 About the Project
 
-🛠 Built With
-html
-& css
 
-Tech Stack
+<!-- TABLE OF CONTENTS -->
 
-Key Features
+# 📗 Table of Contents
 
-🚀 Live Demo
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] While you have made a great effort to include all sections from README [template](https://github.com/microverseinc/readme-template) links to some are broken and some of the sections in the `Table of content` have been deleted from the README. Please make sure to add all sections and their customized detail into the README